gradle/init.gradle.kts

```diff
@@ -1,32 +1,46 @@
-fun resolvePluginPath(rootPath: java.nio.file.Path): java.nio.file.Path? {
-    if (rootPath.resolve("src").toFile().exists()) {
-        return null
-    }
-    if (rootPath.resolve("demo").toFile().exists()) {
-        return null
-    }
-    var path = rootPath
-    for (n in 1..4) {
-        val settingsFile = path.resolve("demo")
-        if (settingsFile.toFile().exists()) {
-            return path.toAbsolutePath()
-            break
-        } else {
-            path = path.resolve("..")
-            println("new path: ${path.toAbsolutePath()}")
-        }
-    }
-    return null
-}
-
 gradle.settingsEvaluated {
```

Comment:
Why is this in an init.gradle rather than a settings.gradle?

Reply:
Demo project is a standalone project demostrating usages. In general case you can copy whole project or a part and use it raw. 

`includeBuild` is... a bit quite complicated to use in such plugin configuration.

By supplying few variables with version, and local maven repository location (`-Dmaven.repo.local=folder`), this `init.gradle.kts` rewrites to use a specific one without project modification. This has been tested with Java and Android demos I have included.